### ISSUE TITLE
Display directories contents 

### ISSUE THREAD

**Issue by ry** [denoland/registry#99]
https://deno.land/std/http/

**Comment by j-f1** [comment]
This would involve somehow fetching the directory contents using a method other than the one used to fetch the contents of requested files. That means that this would probably end up being GitHub-specific since an API call is required.

**Comment by ry** [comment]
@j-f1 Yes, I think we should specially handle github links (at the moment all the repos are from github)

**Comment by j-f1** [comment]
This is done!
